tests/phpunit/tests/test-default-term.php

```diff
@@ -139,4 +139,19 @@ function test_default_category_in_list_table() {
 		$this->assertFalse( strpos( $list, 'edit-tags.php?action=delete&amp;taxonomy=category&amp;tag_ID=' . $fr . '&amp;' ) );
 		$this->assertNotFalse( strpos( $list, 'edit-tags.php?action=delete&amp;taxonomy=category&amp;tag_ID=' . $id . '&amp;' ) );
 	}
+
+	function test_update_term_when_updating_default_language() {
+		$option = get_option( 'default_category' );
+		$option_lang = self::$model->term->get_language( $option );
+
+		$this->assertEquals( 'en', $option_lang->slug );
+
+		$es = self::$model->get_language( 'es' );
+		self::$model->update_default_lang( $es->slug );
```

Comment:
Wouldn't it be simpler to do this?
```suggestion
		self::$model->update_default_lang( 'es' );
```